### PR TITLE
Update appinfo_current.md

### PR DESCRIPTION
**Pull request by jaigak**

### CODE REVIEW

**Comment by Court72** [comment]
@alvinashcraft

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: windows.applicationmodel/appinfo_current.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+
 ## -description
 Retrieves the AppInfo object corresponding to the currently running process.
 
 ## -property-value
 The AppInfo that corresponds to the process.
 
 ## -remarks
-When used by an unpackaged application, this property returns a null pointer.
+When used by an unpackaged application, accessing this property throws HRESULT 80073D54.
```
Comment (alvinashcraft) [inline windows.applicationmodel/appinfo_current.md]:
```suggestion
When used by an unpackaged application, accessing this property throws an **InvalidOperationException** with HRESULT `80073D54`, indicating "The process has no package identity."

```

**Review by alvinashcraft — APPROVED** [review]

**Comment by alvinashcraft** [comment]
#sign-off

File: windows.applicationmodel/appinfo_current.md (hunk 1 above)
Comment (jaigak) [reply windows.applicationmodel/appinfo_current.md]: InvalidOperationException is .NET-specific, while WinRT APIs are for everyone.

**Pull request merged**